Description: Upstream changes introduced in version 1.7.4p4-2.squeeze.3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 sudo (1.7.4p4-2.squeeze.3) stable-security; urgency=high
 .
   * CVE-2012-2337
 .
 The person named in the Author field signed this changelog entry.
Author: Moritz Muehlenhoff <jmm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- sudo-1.7.4p4.orig/match.c
+++ sudo-1.7.4p4/match.c
@@ -627,7 +627,7 @@ addr_matches_if(n)
 	ifp = &interfaces[i];
 	if (ifp->family != family)
 	    continue;
-	switch(family) {
+	switch (family) {
 	    case AF_INET:
 		if (ifp->addr.ip4.s_addr == addr.ip4.s_addr ||
 		    (ifp->addr.ip4.s_addr & ifp->netmask.ip4.s_addr)
